src/bidmanager.js

```diff
@@ -108,11 +108,11 @@ exports.addBidResponse = function (adUnitCode, bid) {
     }
 
     if (!adUnitCode) {
-      utils.logWarn(errorMessage('No adUnitCode was supplied to addBidResponse.'));
```

Comment:
true, these checks really shouldn't be necessary anyway once we have a proper adapter validator in place.